src/components/List.js

```diff
@@ -149,7 +156,15 @@ const List = ({ token }) => {
               <a href="/add-item">Add an Item</a>
             </section>
           ) : (
-            <div>
+            <div className="top-container">
```

Comment:
Looks great and works great. One thing we might want to discuss as a group - in the empty list view, we used an anchor tag for the link to `add item`. Personally, I think the button that you've done here looks way better, but I think we had discovered that the link was the more appropriate html tag in regards to accessibility. So maybe we should talk about that 🤔

Reply:
> Looks great and works great. One thing we might want to discuss as a group - in the empty list view, we used an anchor tag for the link to `add item`. Personally, I think the button that you've done here looks way better, but I think we had discovered that the link was the more appropriate html tag in regards to accessibility. So maybe we should talk about that 🤔

Yea I actually agree. It purpose is to link the user to the Add Item page, not to submit or trigger an event. I can change it to a link. Makes perfect sense.

Reply:
Hey all. I think we could style the button with a background image like [this](https://stackoverflow.com/questions/8683528/embed-image-in-a-button-element)

Reply:
> Hey all. @DeeDowns I think we could style the button with a background image like [this](https://stackoverflow.com/questions/8683528/embed-image-in-a-button-element)

Brittany brought up a great point. For semantic and accessibility a link makes more sense. I agree with her.
What do you think 🤔

Reply:
Could we wrap the button inside an anchor tag?